PocketKit/Sources/Sync/Operations/FetchList.swift

```diff
@@ -93,16 +94,18 @@ class FetchList: SyncOperation {
     }
 
     private func fetchPage(_ pagination: PaginationSpec) async throws -> GraphQLResult<FetchSavesQuery.Data> {
-        let query = FetchSavesQuery(token: token)
-        query.pagination = PaginationInput(
-            after: pagination.cursor,
-            first: pagination.maxItems
+        let query = FetchSavesQuery(
```

Comment:
@bassrock  If this is an example of where you are having the issue with the nil coalescing operator, I think it is because the `PaginationInput` itself should also be wrapped in a `GraphQLNullable`. 

The `pagination` parameter has a type of `GraphQLNullable<PaginationInput>`.

We can’t initialize a `GraphQLNullable` from a literal with a custom object. We do it for literals using the `ExpressibleByStringLiteral`, `ExpressibleByIntegerLiteral`, etc. protocols in Swift, but for our generated objects, there is not a great way to do that. So the solution is to do this:
``` swift
let query = FetchSavesQuery(
  token: token,
  pagination: .some(PaginationInput(
     after: pagination.cursor ?? .none,
     first: pagination.maxItems
  )
)
```

You aren't the first person to be confused by this one though, so I'd love to come up with a better alternative. I'm just not sure what that would be.

Reply:
@AnthonyMDev Yea so we did try this and unfortunately, it still gives us a compiler error with the nill coalescing operator. (Had to also wrap `pagination.maxItems` in `.some()`
  
![Screen Shot 2022-10-20 at 5 15 13 PM](https://user-images.githubusercontent.com/1010384/197082600-b958ee57-12fa-449d-ae15-f225c39761e5.png)